app/build.gradle

```diff
@@ -109,8 +109,15 @@ dependencies {
     compile 'com.github.gfx.android.orma:orma:0.19.0'
     debugCompile 'com.facebook.stetho:stetho:1.2.0'
     debugCompile 'com.facebook.stetho:stetho-okhttp:1.2.0'
-    testCompile 'org.robolectric:robolectric:3.0'
+    testCompile 'com.github.gfx.android.robolectricinstrumentation:robolectric-instrumentation:3.0.5'
     testCompile 'junit:junit:4.12'
     testCompile 'org.hamcrest:hamcrest-library:1.3'
     testCompile 'com.google.guava:guava:18.0'
+    androidTestCompile('com.android.support.test:runner:0.4.1') {
```

Comment:
Note that `Robolectric-Instrumentation` implements the interface of  `'com.android.support.test:runner:0.4.1'`